sdk/tests/conformance/context/context-attributes-alpha-depth-stencil-antialias.html

```diff
@@ -316,10 +317,17 @@
         255, 0, 0, 255,
         255, 0, 0, 255]);
     drawAndReadPixel(gl, vertices, colors, 0, 0);
-    var buf = new Uint8Array(1 * 1 * 4);
-    gl.readPixels(0, 0, 1, 1, gl.RGBA, gl.UNSIGNED_BYTE, buf);
-    pixel[0] = buf[0];
-    shouldBe("pixel[0] != 255 && pixel[0] != 0", "contextAttribs.antialias");
+    var buf_1 = new Uint8Array(1 * 1 * 4);
+    var buf_2 = new Uint8Array(1 * 1 * 4);
+    gl.readPixels(0, 0, 1, 1, gl.RGBA, gl.UNSIGNED_BYTE, buf_1);
+    gl.readPixels(0, 1, 1, 1, gl.RGBA, gl.UNSIGNED_BYTE, buf_2);
+    pixel_1[0] = buf_1[0];
+    pixel_2[0] = buf_2[0];
```

Comment:
Ah darn. Thanks for pointing that out. Yes, it's fine as is. We should really change these primitives to avoid using eval, but that's future work.